src/ifd-nfc.h

```diff
@@ -31,4 +31,8 @@
 #define IFDNFC_SET_ACTIVE_SE     2
 #define IFDNFC_GET_STATUS        3
 
+#ifndef IFD_NFC_OPEN_RETRY_INTERVAL 
```

Comment:
The ``#ifndef`` should not be needed. The header file already has a ``#ifndef _IFD_NFC_H_``